Tests/RxSwiftTests/PrimitiveSequenceTest.swift

```diff
@@ -671,6 +671,52 @@ extension PrimitiveSequenceTest {
             completed(202)
             ])
     }
+    
+    func testSingle_using_producesSingleElement() {
+        let scheduler = TestScheduler(initialClock: 0)
+        
+        var disposeInvoked = 0
+        var createInvoked = 0
+        
+        var disposable: MockDisposable!
+        var xs:TestableObservable<Int>!
```

Comment:
Can you please fix the whitespace to be consistent with the rest of the project.